website/docs/components/form/checkbox/partials/accessibility/accessibility.md

```diff
@@ -18,18 +18,14 @@
 
 `Form::Checkbox::Base` is not conformant until it has an accessible name.
 
-!!! Info
+## Known issues
 
-**Links within labels, help text, or error text**
-
-If a link is used within a label, helper text, or error text, it will not be presented as a link to the user with a screen reader; only the text content is read out. As such, it is generally preferable to avoid links within help/error text or labels; however, we understand that this may not be avoidable in some cases. Please use sparingly until a good known alternative approach is determined.
-!!!
+If a link is used within a label, helper text, or error text, it will not be presented as a link to the user with a screen reader; only the text content is read out. As such, care should be used when considering this feature.
```

Comment:
```suggestion
If a link is used within a label, helper text, or error text, it will not be presented as a link to the user with a screen reader; only the text content is read out. As such, care should be used if considering this approach.
```